be/src/main/java/com/secondhand/user/UserController.java

```diff
@@ -16,14 +17,23 @@
 public class UserController {
 
     private final UserService userService;
-    private final JwtUtil jwtUtil;
 
-    @PutMapping("/regions")
-    public ResponseEntity<CustomResponse> updateMyRegion(@RequestBody PostMyRegionDto postMyRegionDto, @RequestHeader("Authorization") String bearerToken) {
+    @GetMapping("/regions")
+    public ResponseEntity<CustomResponse<UserRegionsDto>> getMyRegion(@RequestAttribute LoggedInUser loggedInUser) {
 
-        // TODO: 토근 처리
+        return ResponseEntity
+                .ok()
+                .body(new CustomResponse(
+                        "success",
+                        200,
+                        "지역 조회 성공",
+                        userService.getMyRegion(loggedInUser.getId())));
+    }
```

Comment:
```suggestion
        UserRegionDto userRegionDto = userService.getMyRegion(loggedInUser.getId());
        
        return ResponseEntity
                .ok()
                .body(new CustomResponse(
                        "success",
                        200,
                        "지역 조회 성공",
                        userRegionDto));
    }
```

어느 코드가 더 가독성이 좋아보이시나요?